lldb/test/API/functionalities/process_save_core_minidump/TestProcessSaveCoreMinidump.py

```diff
@@ -67,6 +67,18 @@ def verify_core_file(
             self.assertIn(thread_id, stacks_to_registers_map)
             register_val_list = stacks_to_registers_map[thread_id]
             frame_register_list = frame.GetRegisters()
+            # explicitly verify we collected fs and gs base for x86_64
```

Comment:
Don't we need some logic here to do this only for x86_64 core files? Or is this test somehow x86_64 only? I didn't see anything in the test that made this x86_64 specific